.travis.yml

```diff
@@ -13,7 +13,9 @@ rvm:
 script:
     - bundle exec rails db:reset db:setup db:migrate RAILS_ENV=test
     - bundle exec rspec
-    
+    - git push heroku master
+    - heroku run rake db:migrate
+
```

Comment:
It seems like this tried to push to heroku when the pull request was sent. What we actually want is for it to push to heroku AFTER the PR is merged into master. Is there a thing called 'after script'?

Reply:
I removed this (since it didn't work anyway)- it was an initial attempt and I have done more research on how to implement this functionality.